sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -43,13 +52,15 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
     assessmentSequentialProcessingService = nestApplication.get(
       AssessmentSequentialProcessingService,
     );
+    auditUser = systemUsersService.systemUser;
+    now = new Date();
   });
 
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.restoreAllMocks();
```

Comment:
👍

Reply:
Thanks for investigating the issue with me. 😃